Dockerfile

```diff
@@ -13,10 +13,9 @@
 # limitations under the License.
 
 FROM golang:1.13-buster AS build
-WORKDIR /go/src/github.com/vmware-tanzu/velero-plugin-for-aws
-# copy vendor in separately so the layer can be cached if the contents don't change
-COPY vendor vendor
-COPY velero-plugin-for-aws velero-plugin-for-aws
+COPY . /go/src/velero-plugin-for-aws
+WORKDIR /go/src/velero-plugin-for-aws
+RUN go mod download
```

Comment:
is there a specific reason to include this step?

Reply:
No, removed.